style.css

```diff
@@ -0,0 +1,153 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;700&display=swap');
+
+body {
+  margin: auto;
+  width: 100%;
+  max-width: 375px;
+}
```

Comment:
- [ ] there is empty space (see below image) on the sides of the application, you can fix this by changing the `margin` to "0" and removing the `max-width`
![Screenshot 2022-03-21 111303](https://user-images.githubusercontent.com/39240820/159241324-ed4d40cc-8b76-4d88-9789-daef2aae5a65.png)